code/controllers/subsystems/traumas.dm

```diff
@@ -0,0 +1,125 @@
+var/datum/controller/subsystem/traumas/SStraumas
+
+/datum/controller/subsystem/traumas
+	name = "Traumas"
+	flags = SS_NO_FIRE
+	init_order = SS_INIT_MISC
+	var/list/phobia_types
+	var/list/phobia_words
+	var/list/phobia_mobs
+	var/list/phobia_objs
+	var/list/phobia_turfs
+	var/list/phobia_species
+
+#define PHOBIA_FILE "phobia.json"
+
+/datum/controller/subsystem/traumas/New()
+    NEW_SS_GLOBAL(SStraumas)
+
+/datum/controller/subsystem/traumas/Initialize()
+	phobia_types = list("spiders", "space", "security", "doctors", "clowns", "lizards", "cats", "humans", "skrell", "robots", "pests", "nanotrasen", "filth", "paranormals", "death", "darkness")
+
+	phobia_words = list("spiders"   = strings(PHOBIA_FILE, "spiders"),
+						"space"     = strings(PHOBIA_FILE, "space"),
+						"security"  = strings(PHOBIA_FILE, "security"),
+						"clowns"    = strings(PHOBIA_FILE, "clowns"),
+						"doctors"= strings(PHOBIA_FILE, "doctors"),
+						"lizards"    = strings(PHOBIA_FILE, "unathi"),
+						"cats"       = strings(PHOBIA_FILE, "cats"),
+						"humans"    = strings(PHOBIA_FILE, "humans"),
+						"skrell"    = strings(PHOBIA_FILE, "skrell"),
+						"robots"    = strings(PHOBIA_FILE, "robots"),
+						"pests"     = strings(PHOBIA_FILE, "pests"),
+						"nanotrasen"= strings(PHOBIA_FILE, "nanotrasen"),
```

Comment:
This seems too narrow. Maybe you should change it to megacorps in general, which seems much more reasonable... in comparison, at least.

Reply:
On second thought, combining security and NanoTrasen into "authority" would make much more sense.

Reply:
The distinction as it is here works well. Including other megacorps would be pointless as they have little mechanical representation.

Reply:
Prosthetics, for example? It also opens the way for having more mechanical representation of other megacorps, too.

Reply:
Putting the cart before the horse there - there's nothing preventing additional mechanical rep of megacorps so this wouldn't pave anything. Fear of prosthetics themselves aren't a terrible idea however.